test/test_speccheck.py

```diff
@@ -1021,7 +1078,7 @@ def test_check_patch_not_applied_not_enforced(package, speccheck):
 # TODO: Add specfile-error test.
 
 
-@pytest.mark.parametrize('package', ['spec/SpecCheck2'])
+@pytest.mark.parametrize('package', ['spec/invalid-url'])
```

Comment:
This individual file solves the error posed in opensuselp15 build [here](https://travis-ci.org/github/rpm-software-management/rpmlint/jobs/704275990#L1655)